assets/style.css

```diff
@@ -19,6 +19,48 @@ body {
   font-family: roboto, arial, sans-serif;
 }
 
+#toTopBtn {
+  position: fixed;
+ bottom: 26px;
+ right: 39px;
+ z-index: 98;
+ padding: 21px;
+ background-color: #000;
+ border-radius: 25%;
+}
+
+.js .cd-top--fade-out {
```

Comment:
You should remove all the remaining CSS you added here, from this line 32 to line 63. None of it is doing anything at all, and there are vars that aren't even declared in this project. The only thing you need from all of this section is the background in `.cd-top` which is needed for the arrow. Move it above to the main element styles `#toTopBtn`
```css
background: url(https://res.cloudinary.com/dxfq3iotg/image/upload/v1571057658/cd-top-arrow.svg) no-repeat center 50%;
```